refactoring.php

```diff
@@ -2,50 +2,49 @@
 
 public function post_confim() {
     $id = Input::get('service_id');
-    $servicio = Service::find($id);
-    //dd($servicio)
-    if ($servicio != NULL) {
-        if ($servicio->status_id == '6') {
-            return Response::json(array('error' => '2'));
-        }
-        if ($servicio->driver_id == NULL && $servicio->status_id == '1') {
-            $servicios = Service::update($id, array(
-                         'driver_id' => Input::get('driver_id'),
-                         'status_id' => '2'
-                            //Up Carro
-                            //,'pwd' => md5(Input::get('pwd'))
-            ));
-            Driver::update(Input::get('driver_id'), array(
-                "available" => '0'
-            ));
-            $driverTmp = Driver::find(Input::get('driver_id'));
-            Service::update($id, array(
-                'car_id' => $driverTmp->car_id
-            ));
-            //Notificar a usuario!
-            $pushMessage = 'Tu servicio ha sido confirmado!';
-            /*  $servicio = Service::find($id);
```

Comment:
Avoid upload coded comment to the repository.
If we want to see the history only we made git log